arduino-ide-extension/src/browser/dialogs/certificate-uploader/select-board-components.tsx

```diff
@@ -44,22 +45,29 @@ export const SelectBoardComponent = ({
       'arduino/certificate/selectBoard',
       'Select a board...'
     );
+    const updatableBoards = boardList.boards.filter((item) => {
+      const fqbn = (
+        isInferredBoardListItem(item) ? item.inferredBoard : item.board
+      ).fqbn;
+      return fqbn && updatableFqbns.includes(fqbn);
+    });
     let selBoard = -1;
-    const updatableBoards = availableBoards.filter(
-      (board) => board.port && board.fqbn && updatableFqbns.includes(board.fqbn)
-    );
-    const boardsList: BoardOption[] = updatableBoards.map((board, i) => {
-      if (board.selected) {
+
+    const boardsList: BoardOption[] = updatableBoards.map((item, i) => {
```

Comment:
here were have a variable name `boardsList` very similar to `boardList` in the same scope, it might make things a bit clearer to rename it more explicitly